src/app/settings/views/Storage/StorageForm/StorageFormFields/StorageFormFields.test.tsx

```diff
@@ -63,18 +63,21 @@ describe("StorageFormFields", () => {
         </MemoryRouter>
       </Provider>
     );
-    wrapper.find("select[name='default_storage_layout']").simulate("change", {
-      target: { name: "default_storage_layout", value: "blank" },
-    });
-    await waitForComponentToPaint(wrapper);
-    expect(wrapper.find(".p-form-validation__message").text()).toBe(
-      "Caution: You will not be able to deploy machines with this storage layout. Manual configuration is required."
+
+    await userEvent.selectOptions(
+      screen.getByRole("combobox", { name: "Default storage layout" }),
+      "No storage (blank) layout"
     );
+
+    // Writing a matcher function for getByText turned out to be too complex here, using a test ID was the only sensible option
+    await waitFor(() => {
+      expect(screen.getByTestId("blank-layout-warning")).toBeInTheDocument();
```

Comment:
Is this because of white space? Have you tried a partial match or `expect(screen.getByTestId("blank-layout-warning")).toHaveTextContent(...)`?

Reply:
I was a bit iffy on it to be honest. It was a whitespace issue in a sense, the piece of text we're looking for is actually broken up between multiple HTML tags. I did try a partial match but wasn't settled on it, in the event that only some of the text actually renders for some reason